core/benches/shredder.rs

```diff
@@ -26,6 +42,21 @@ fn bench_shredder(bencher: &mut Bencher) {
     })
 }
 
+#[bench]
+fn bench_shredder_large_entries(bencher: &mut Bencher) {
+    let kp = Arc::new(Keypair::new());
+    let shred_size = PACKET_DATA_SIZE - *SIZE_OF_DATA_SHRED_HEADER;
+    let num_shreds = ((1000 * 1000) + (shred_size - 1)) / shred_size;
+    let txs_per_entry = 128;
+    let num_entries = max_entries_per_n_shred(&make_test_entry(txs_per_entry), num_shreds as u64);
+    let entries = make_large_unchained_entries(txs_per_entry, num_entries);
+    // 1Mb
```

Comment:
Is this 1mb? I think this comment is unnecessary now.

Reply:
this line `let num_shreds = ((1000 * 1000) + (shred_size - 1)) / shred_size;` should approximate 1MB worth of shreds